data/Templates/Hl7v2/Resource/_Condition.liquid

```diff
@@ -28,6 +28,12 @@
                 },
             ],
         },
+        "asserter":
+        {
+            {% if DG1.16 and Practitioner_ID_DG1_16 -%}
```

Comment:
Why do we need to check both of them? It seems only the latter one is used below.

Reply:
Because we want to keep the reference properties consistent with other properties in the same level.
Like the `location.location.reference` and `location.status`.

Reply:
By offline discussion, we are going to keep this for future refinement of conditions, like moving them to higher level of the tree. Besides, we need to track a item about using properties in references, like `Specimen.receivedTime/{{ Specimen_receivedTime_ID_OBR_13 }}` and `Practitioner|PractitionerRole/{{ Practitioner|PractitionerRole_ID_PV2_13 }}`.